zerver/worker/queue_processors.py

```diff
@@ -427,6 +430,25 @@ def consume(self, event):
 class OutgoingWebhookWorker(QueueProcessingWorker):
     def consume(self, event):
         # type: (Mapping[str, Any]) -> None
-
-        # TODO: Implement this worker.
-        pass
+        message = event['message']
+        realm = get_realm_by_email_domain(message['sender_email'])
+        bot_user = get_user_profile_by_id(event['user_profile_id'])
+        services = get_realm_bot_services(bot_user.email, realm)
+        rest_operation = {'method': 'POST',
+                          'relative_url_path': '',
+                          'request_kwargs': {},
+                          'base_url': ''}
```

Comment:
It is a convenient method though to pass url and other values into doRestCall. We can omit it for now but it maybe useful in future.